mkt/webapps/management/commands/process_addons.py

```diff
@@ -23,41 +18,11 @@
                                               mkt.STATUS_PUBLIC,
                                               mkt.STATUS_APPROVED],
                                   disabled_by_user=False)]},
-    'add_uuids': {'method': add_uuids,
-                  'qs': [Q(guid=None), ~Q(status=mkt.STATUS_DELETED)]},
     'update_supported_locales': {
         'method': update_supported_locales,
         'qs': [Q(disabled_by_user=False,
                  status__in=[mkt.STATUS_PENDING, mkt.STATUS_PUBLIC,
                              mkt.STATUS_APPROVED])]},
-    'dump_apps': {'method': dump_apps,
-                  'qs': [Q(status=mkt.STATUS_PUBLIC, disabled_by_user=False)],
-                  'pre': clean_apps,
-                  'post': zip_apps},
-    'fix_missing_icons': {'method': fix_missing_icons,
-                          'qs': [Q(status__in=[mkt.STATUS_PENDING,
-                                               mkt.STATUS_PUBLIC,
-                                               mkt.STATUS_APPROVED],
-                                   disabled_by_user=False)]},
-    'populate_is_offline': {
```

Comment:
One-shot command used to populate the `is_offline` field when we introduced it in https://bugzilla.mozilla.org/show_bug.cgi?id=1049770. AFAIK we haven't used it since.